app/controllers/think_feel_do_dashboard/arms_controller.rb

```diff
@@ -7,6 +7,7 @@ class ArmsController < ApplicationController
 
     # GET /think_feel_do_dashboard/arms
     def index
+      @arms = Arm.includes(:groups)
```

Comment:
@gwensmu given https://github.com/CanCanCommunity/cancancan/wiki/authorizing-controller-actions, if you are doing this, it seems you may have to say ```load_and_authorize_resource, except: [:index]``` and then augment the index method as so:
```
def index
  authorize! :index, Arm
  @arms = Arm.includes(:groups)
end
```
Or you can create a ```before_filter``` and load the ```@arms``` and then you can keep ```load_and_authorize_resource``` the way it is.  However, you will still have to add ```authorize! :index, Arm```
I believe that is correct, but let me know if you have questions.